TargetTopic:                '/move_base_simple/goal'

VehicleMass:                1.0

GravAcc:                    9.8

HorizDrag:                  0.70

VertDrag:                   0.80

ParasDrag:                  0.01

SpeedEps:                   0.0001

SmoothingEps:               1.0e-2

MaxVelBox:                  4.0

MaxAccBox:                  6.0

ConstRes:                   20

OptOrder:                   3 # 3 for jerk, 4 for snap